src/models/User.js

```diff
@@ -1,10 +1,74 @@
 export default (sequelize, DataTypes) => {
   const User = sequelize.define('user', {
-    firstName: DataTypes.STRING,
-    lastName: DataTypes.STRING,
-    email: DataTypes.STRING,
-    isVerified: DataTypes.BOOLEAN,
-    password: DataTypes.STRING
+    firstName: {
+      allowNull: true,
```

Comment:
I don't think we should allow this field to be nullable since signing up fills it as a requirement.